src/app/static/src/app/components/downloadResults/DownloadResults.vue

```diff
@@ -41,24 +41,33 @@
                         <span class="font-weight-bold" v-translate="'uploadComplete'"></span>
                     </div>
                     <div class="d-flex align-items-center height-40">
-                        <tick color="#e31837" v-if="uploadComplete" width="20px"></tick>
+                        <tick color="#e31837" v-if="uploadComplete"></tick>
+                    </div>
+                </div>
+                <div id="releaseCreated" v-if="releaseCreated || releaseFailed" class="d-flex align-items-end">
+                    <div class="d-flex align-items-center height-40 mr-1">
+                        <span class="font-weight-bold" v-translate="releaseCreated ? 'releaseCreated' : 'releaseFailed'"></span>
+                    </div>
+                    <div class="d-flex align-items-center height-40">
+                        <component :is="releaseCreated ? 'tick' : 'cross'" color="#e31837"></component>
                     </div>
                 </div>
                 <error-alert v-if="uploadError" :error="uploadError"></error-alert>
             </div>
         </div>
-        <upload-modal id="upload-modal" :open="uploadModalOpen" @close="uploadModalOpen = false"></upload-modal>
+        <upload-modal id="upload-modal" v-if="uploadModalOpen" :open="uploadModalOpen" @close="uploadModalOpen = false"></upload-modal>
```

Comment:
Interested to know why you've added the `v-if` here, in addition to `:open`. Would normally do the latter - having both looks a bit redundant?

Reply:
To avoid having to watch the uploadFile
![image](https://user-images.githubusercontent.com/57633087/134479815-28ce5f01-c662-4ee3-9f6a-f465b0c76f47.png)

Reply:
The v-if was to avoid the modal needing to include some tedious watch on values (the upload files themselves I think), which would always be set whenever it was opened, but not if the component was mounted closed immediately. So I guess `:open` could just be `"true"` if we're rendering based on `uploadModalOpen`